pkg/ipam/cell/cell.go

```diff
@@ -70,7 +71,7 @@ type ipamAPIHandlerOut struct {
 }
 
 func newIPAMAPIHandler(params ipamAPIHandlerParams) ipamAPIHandlerOut {
-	if option.Config.LoadBalancerOnly {
+	if !params.Clientset.IsEnabled() {
```

Comment:
This does not look correct to me. IPAM is used even without K8s, e.g. via the docker plugin

Reply:
@gandro is there any way where we can detect that docker plugin is running to feed this into the above? I want to remove the LoadBalancerOnly conditional ideally.

Reply:
There might also be other users of the API outside of the docker plugin. It's an API after all, we don't know who is using it to manually create endpoints.

I think the old approach of disabling it based on a specific flag that the user has to explicitly opt-in is the best appraoch. Though I'm not sure why someone would want to disable it, it shouldn't impact other subsystems after all, since IPAM is still active even if you disable the API.